MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -1192,6 +1192,97 @@
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_5x5_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_5x5_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_6x5_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_6x5_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_6x6_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_6x6_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_8x5_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_8x5_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_8x6_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_8x6_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_8x8_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_8x8_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_10x5_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_10x5_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_10x6_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_10x6_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_10x8_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_10x8_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_10x10_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_10x10_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_12x10_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_12x10_sRGB, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_12x12_LDR, RF );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily1_v1, ASTC_12x12_sRGB, RF );
+
+	addMTLPixelFormatCapabilities( tvOS_GPUFamily2_v1, Depth32Float, DRMR );
+	addMTLPixelFormatCapabilities( tvOS_GPUFamily2_v1, Depth32Float_Stencil8, DRMR );
+
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily2_v1, BGRA10_XR, All );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily2_v1, BGRA10_XR_sRGB, All );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily2_v1, BGR10_XR, All );
+	addMTLPixelFormatCapabilities(tvOS_GPUFamily2_v1, BGR10_XR_sRGB, All );
+
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, UCharNormalized, Vertex );
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, CharNormalized, Vertex );
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, UChar, Vertex );
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, Char, Vertex );
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, UShortNormalized, Vertex );
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, ShortNormalized, Vertex );
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, UShort, Vertex );
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, Short, Vertex );
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, Half, Vertex );
+	addMTLVertexFormatCapabilities( tvOS_GPUFamily1_v1, UChar4Normalized_BGRA, Vertex );
```

Comment:
I have simply copied over all the `addMTLVertexFormatCapabilities` from iOS and set them as available starting `GPUFamily1_v1`.

Reply:
Yes it does compile. I see tvOS listed on https://developer.apple.com/documentation/metal/mtlvertexformat/mtlvertexformatucharnormalized?language=objc

Reply:
The required feature set should probably be `tvOS_GPUFamily1_v3`, since these were only introduced with Metal 2.0 (tvOS 11).